src/locale/locales/de/messages.po

```diff
@@ -830,7 +792,7 @@ msgstr "Verbinden..."
 
 #: src/view/com/auth/create/CreateAccount.tsx:209
 msgid "Contact support"
-msgstr ""
+msgstr "Kontakt zum Support"
```

Comment:
Hard without context, but in case this "Contact" is meant to be a verb, then use

```suggestion
msgstr "Support kontaktieren"
```

Reply:
Done 👍 (I believe this is a link for when people are having trouble during the process of creating an account)